src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp

```diff
@@ -165,7 +164,17 @@ void init_button(JingleButton *button)
 	MenuController::addAction<JContactResource>(button);
 }
 
-Q_GLOBAL_STATIC_WITH_INITIALIZER(JingleButton, button, init_button(x.data()))
+//Q_GLOBAL_STATIC_WITH_INITIALIZER(JingleButton, button, init_button(x.data()))
+
+JingleButton *button()
```

Comment:
May you consider creating button in JingleSupport's constructor and deleting in JingleSupport's destructor or do something like this?